helm-chart/renku/values.yaml

```diff
@@ -1643,4 +1643,4 @@ affinity: {}
 versions:
   latest:
     image:
-      tag: v2.9.2
+      tag: v2.9.4
```

Comment:
This whole section is not needed. It is leftover from an old version of the helm chart. I cannot find .Values.versions being used anywhere in the helm chart.

And github for some reason will not allow me to do a suggestion here. You can just remove `versions` and anything inside of it.